### ISSUE TITLE
blank name

### ISSUE THREAD

**Issue by Sorunome** [toxygen-project/toxygen#24]
If one of your contacts has a blank name / description then, when switching to that contact, it still displays the avatar / name / description of the previous contact

EDIT: It might have to do with it that said contact was new?

Restarting toxygen resulted in
```
Traceback (most recent call last):
  File "/opt/toxygen/toxygen/main.py", line 462, in <module>
    main()
  File "/opt/toxygen/toxygen/main.py", line 458, in main
    toxygen.main()
  File "/opt/toxygen/toxygen/main.py", line 184, in main
    self.ms = MainWindow(self.tox, self.reset, self.tray)
  File "/opt/toxygen/toxygen/mainscreen.py", line 20, in __init__
    self.initUI(tox)
  File "/opt/toxygen/toxygen/mainscreen.py", line 326, in initUI
    self.profile = Profile(tox, self)
  File "/opt/toxygen/toxygen/profile.py", line 63, in __init__
    self._history.add_friend_to_db(tox_id)
  File "/opt/toxygen/toxygen/history.py", line 69, in add_friend_to_db
    cursor.execute('CREATE TABLE id' + tox_id + '('
sqlite3.OperationalError: database is locked
```

I had to delete the histroy files to get it to start again

**Comment by ingvar1995** [comment]
History bug is known and has nothing with empty name

**Comment by ingvar1995** [comment]
Hm, seems that problem is in history, Toxygen processes empty names well.

**Comment by ingvar1995** [comment]
Hope it was fixed in 42e0ec0
